server/content-types/image/schema.json

```diff
@@ -32,18 +32,27 @@
         "type": "text"
       }
     },
-    "mediaId": {
-      "type": "integer"
-    },
     "timestamp": {
       "type": "datetime"
     },
     "caption": {
-      "type": "text",
```

Comment:
Hi, @SvenWesterlaken you missing column in this row. It's causing error while run.

Reply:
Fixed in my last commit, thanks for mentioning @zadinvit